services/spar/test-integration/Test/Spar/DataSpec.hs

```diff
@@ -265,7 +268,7 @@ testDeleteTeam = it "cleans up all the right tables after deletion" $ do
   ssoid1 <- getSsoidViaSelf (getUid storedUser1)
   ssoid2 <- getSsoidViaSelf (getUid storedUser2)
   -- Delete the team
-  runSparCass $ Data.deleteTeam tid
+  runSpar $ App.deleteTeam tid
```

Comment:
No, `App.deleteTeam` does something different from `Data.deleteTeam`.

Reply:
`deleteTeam` moved from `Spar.Data` to `Spar.App` in this PR :)

Reply:
Oops, sorry, my bad.